README.md

```diff
@@ -1,2 +1,31 @@
-# hw01
```

Comment:
Evaluation HW-01 for Harry Lee
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   NA        |              |            |
| **Coding strategy**         |     NA      |              |            |
| **Presentation**            |     I think it's great! I like the roasted duck photos, and it is impressive that you have worked in Caixin, one of my most favorite medias. You have used the required syntax for Markdown well.  One suggestion I will give is that it is a little weird to make author as a little heading. You can either add some content, or change your interests into little heading, or separate your interests and  your prior experience into different sections.   |              |            |
| **Achievement, creativity** |    All is good besides the heading issue. Also, not sure whether a bio need your own photo.       |              |            |
| **Ease of access**          |    NA       |              |            |